composedtaskrunner-task-app-dependencies/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>composedtaskrunner-task-app-dependencies</artifactId>
+    <version>1.1.1.BUILD-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Composed Task Runner Task App Dependencies</name>
+    <description>Dependencies for Spring Cloud Task Composed Task Runner App Starter</description>
+
+	<properties>
+		<spring-cloud-task.version>1.1.2.RELEASE</spring-cloud-task.version>
+		<spring-cloud-dataflow-rest-client.version>1.2.0.BUILD-SNAPSHOT</spring-cloud-dataflow-rest-client.version>
+	</properties>
+    <parent>
+        <groupId>org.springframework.cloud.task.app</groupId>
+        <artifactId>task-app-starters-core-dependencies</artifactId>
+        <version>1.1.1.BUILD-SNAPSHOT</version>
+    </parent>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.cloud.task.app</groupId>
+                <artifactId>spring-cloud-starter-task-composedtaskrunner</artifactId>
+                <version>1.1.1.BUILD-SNAPSHOT</version>
+            </dependency>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-task-core</artifactId>
+				<version>${spring-cloud-task.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dataflow-rest-client</artifactId>
+				<version>${spring-cloud-dataflow-rest-client.version}</version>
+			</dependency>
+		</dependencies>
+    </dependencyManagement>
+<profiles>
+		<profile>
+			<id>spring</id>
+			<repositories>
+				<repository>
+					<id>spring-snapshots</id>
+					<name>Spring Snapshots</name>
+					<url>http://repo.spring.io/libs-snapshot-local</url>
+					<snapshots>
+						<enabled>true</enabled>
+					</snapshots>
+				</repository>
+				<repository>
+					<id>spring-milestones</id>
+					<name>Spring Milestones</name>
+					<url>http://repo.spring.io/libs-milestone-local</url>
+					<snapshots>
+						<enabled>false</enabled>
+					</snapshots>
+				</repository>
+				<repository>
+					<id>spring-releases</id>
+					<name>Spring Releases</name>
+					<url>http://repo.spring.io/release</url>
+					<snapshots>
+						<enabled>false</enabled>
+					</snapshots>
+				</repository>
+				<repository>
+					<id>spring-libs-release</id>
+					<name>Spring Libs Release</name>
+					<url>http://repo.spring.io/libs-release</url>
+					<snapshots>
+						<enabled>false</enabled>
+					</snapshots>
+				</repository>
```

Comment:
`repo.spring.io/release` is there mainly to pull any libs that hasn't reached maven central yet. For instance, we do an app generator maven plugin release, but it hasn't synched with central yet, and we need to make a release for apps. I believe thats the primary reason whey we put it there.